.github/workflows/test-monai-deploy-express.yml

```diff
@@ -0,0 +1,41 @@
+# This is a basic workflow to help you get started with Actions
+
+name: Test MONAI Deploy Express
+on:
+  push:
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+# Controls when the workflow will run
+
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "build"
+  build:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v3
+      
+      - name: Start MONAI Deploy Express
+        working-directory: ./deploy/monai-deploy-express
+        run: docker compose up -d --wait
+        
+      - name: Informatics Gateway Health Check
+        continue-on-error: true
```

Comment:
Just checking that you intended to leave the continue-on-error flag? I guess it won't affect the actual test result.

Reply:
Might be good to keep it to test other services?